open-metadata-implementation/adapters/governance-engines-plugins/gaian-ranger-plugin/pom.xml

```diff
@@ -5,306 +5,203 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <!-- About this module -->
     <version>0.3-SNAPSHOT</version>
     <artifactId>gaian-ranger-plugin</artifactId>
     <name>Ranger Gaian Plugin</name>
     <description>Ranger Gaian Plugin</description>
     <packaging>jar</packaging>
 
-    <!-- Parent project -->
     <parent>
         <groupId>org.odpi.egeria</groupId>
         <version>0.3-SNAPSHOT</version>
         <artifactId>governance-engines-plugins</artifactId>
     </parent>
 
+    <properties>
```

Comment:
This is an improvement :-)

But ultimately we should move all the versions up to the top level pom
Can either do here or track in a new issue (as it's a step forward!)

Reply:
Can we track this in a new issue? Also, the adapter pom level for these dependencies could not be the best solution.

Reply:
tracking in https://github.com/odpi/egeria/issues/443